Firebase/Core/Private/FIRLibrary.h

```diff
@@ -35,7 +35,9 @@ NS_SWIFT_NAME(Library)
 @optional
 /// Implement this method if the library needs notifications for lifecycle events. This method is
 /// called when the developer calls `FirebaseApp.configure()`.
-+ (void)configureWithApp:(FIRApp *)app;
++ (void)configureWithApp:(FIRApp *)app
```

Comment:
I'm fairly certain this isn't used anymore, but we can leave it for a release or two just to be sure.

Reply:
Technically, if Firebase 6 pods use it, it should not be deleted until Firebase 7.

Reply:
Makes sense, that's fine with me :)

Reply:
Maybe it should be deleted since the implementation is gone.  Also looks like all usages are in this repo.

Reply:
All usages are in this repo, sure, but we still call it on any SDKs that register and implement that method. (in `+[FIRApp sendNotificationToSDKs:]`). I think it makes sense to keep it around until next breaking change just in case folks get a new Core SDK with an older SDK that still depends on it.